Model/Order/OrderAddress.php

```diff
@@ -0,0 +1,80 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace ScandiPWA\SalesGraphQl\Model\Order;
+
+use Magento\Sales\Api\Data\OrderAddressInterface;
+use Magento\Sales\Api\Data\OrderInterface;
+use Magento\SalesGraphQl\Model\Order\OrderAddress as CoreOrderAddress;
+
+/**
+ * Class to get the order address details
+ */
+class OrderAddress extends CoreOrderAddress
+{
+    /**
+     * Get the order Shipping address
+     *
+     * @param OrderInterface $order
+     * @return array|null
+     */
+    public function getOrderShippingAddress(
+        OrderInterface $order
+    ): ?array {
```

Comment:
I assume we're safe at setting 7.4+ based php code on all updates, now? (specially considering new java vulnerability)